packages/vm/lib/evm/types.ts

```diff
@@ -0,0 +1 @@
+export type Log = [Buffer, Buffer[], Buffer]
```

Comment:
great PR! unfortunately we are not on TypeScript 4.0 yet where we can [label tuple elements](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements) but meanwhile I think this would be helpful:

```suggestion
export type Log = [Buffer, Buffer[], Buffer] // [address, topics, data]
```